internal/block/block.go

```diff
@@ -115,19 +148,3 @@ func createBlock(blockSize int64) (Block, error) {
 	}
 	return &mb, nil
 }
-
-// ReadAt reads data from the block at the specified offset.
```

Comment:
For review: this method is currently unused and missed in this PR: https://github.com/GoogleCloudPlatform/gcsfuse/pull/3503